### PR TITLE
feat(VItemGroup): add tag prop to component

### LINKED ISSUE #11392: [Feature Request] Allow changing the tag for VBottomNav
### Problem to solve
Screen readers like having the main site navigation in a `<nav>` element. See https://dequeuniversity.com/rules/axe/3.5/region . This is also good semantic HTML5 practice.

### Proposed solution
Some Vuetify components allow changing their root tag from `div` to something else through the `tag` prop. VBottomNav should also support that prop.

### PR DESCRIPTION
**Pull request by ElijahKotyluk**
## Description

resolves #11392 

## How Has This Been Tested?

Jest | playground
## Markup:

<details>

```vue
<template>
  <v-item-group tag="footer">
    <v-container>
      <v-row>
        <v-col
          v-for="n in 3"
          :key="n"
          cols="12"
          md="4"
        >
          <v-item v-slot="{ active, toggle }">
            <v-card
              :color="active ? 'primary' : ''"
              class="d-flex align-center"
              dark
              height="200"
              @click="toggle"
            >
              <v-scroll-y-transition>
                <div
                  v-if="active"
                  class="display-3 flex-grow-1 text-center"
                >
                  Active
                </div>
              </v-scroll-y-transition>
            </v-card>
          </v-item>
        </v-col>
      </v-row>
    </v-container>
  </v-item-group>
</template>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by jacekkarczmarczyk** [comment]
`tag` may be misleading - it suggest that it's the tag for component itself, not for the item, maybe `item-tag` would be better?

Also please provide playground for your PRs. Thanks

**Comment by KaelWD** [comment]
> it suggest that it's the tag for component itself, not for the item

It is for the component though?

File: packages/api-generator/src/maps/v-item-group.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 module.exports = {
   'v-item-group': {
+    props: [
```
Comment (KaelWD) [inline packages/api-generator/src/maps/v-item-group.js:3]: Was this not being picked up automatically?

**Review by KaelWD — APPROVED** [review]

**Comment by jacekkarczmarczyk** [comment]
> It is for the component though?

Ah yeah, i guess i was mislead by some other playground or other piece of code

**Pull request merged**